.circleci/config.yml

```diff
@@ -364,6 +364,8 @@ jobs:
     executor:
       name: go_cimg
     steps:
+      - docker/install-docker
```

Comment:
maybe we could try to add 
`- setup_remote_docker`
after line 367, to see if with that docker is running.